
config4 Multi
Traceback (most recent call last):
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 105, in <module>
    confusion()
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 75, in confusion
    model.load_state_dict(torch.load(save_path))
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for ResNet18_5_Multi:
	size mismatch for model.conv1.weight: copying a param with shape torch.Size([64, 4, 7, 7]) from checkpoint, the shape in current model is torch.Size([64, 5, 7, 7]).